vpr/src/pack/noc_aware_cluster_util.h

```diff
@@ -0,0 +1,13 @@
+
+#ifndef VTR_NOC_AWARE_CLUSTER_UTIL_H
```

Comment:
Comment purpose of overall file. This would be a logical place for a high-level comment on the whole algorithm / idea and a ref to the FPL paper.